### PR TITLE
Save choosen blocks level

### PR DESCRIPTION
**Pull request by tavriaforever**
// cc @gela-d

### CODE REVIEW

File: src/blocks/common.blocks/menu-list/_type/menu-list_type_level.js [hunk 1]
```diff
@@ -29,8 +38,27 @@ modules.define('menu-list', ['i-bem__dom', 'jquery'], function(provide, BEMDOM,
...
+
+        /**
+         * Saves the passed level to window.localStorage
+         * @param level
+         * @private
+         */
+        _saveLevelToStorage: function(level) {
+            var localStorage = window.localStorage;
+
+            if(localStorage.getItem('level') !== level) localStorage.setItem('level', level);
```
Comment (tadatuta) [inline src/blocks/common.blocks/menu-list/_type/menu-list_type_level.js]: i'd remove the check

**Comment by gela-d** [comment]
Everything works

**Comment by gela-d** [comment]
need port this pr into leveldb-tree

**Comment by tavriaforever** [comment]
Clone commit into leveldb https://github.com/bem/bem-site-engine/commit/69c94c58223779045daf28b0c423b0d19eedbc81

**Pull request merged**